src/pages/Admin/Admin.tsx

```diff
@@ -277,10 +277,16 @@ const EventsComponent: React.FC<VenueDetailsPartProps> = ({ venue }) => {
     },
   ]);
 
-  const events = useSelector((state) => state.firestore.ordered.events);
+  let events = useSelector((state) => state.firestore.ordered.events);
   const [showCreateEventModal, setShowCreateEventModal] = useState(false);
   const [showDeleteEventModal, setShowDeleteEventModal] = useState(false);
   const [editedEvent, setEditedEvent] = useState<WithId<VenueEvent>>();
+  events?.sort((a, b) => a.start_utc_seconds - b.start_utc_seconds);
```

Comment:
This can be done at the firebase level, would be better.

Reply:
It seems to be the case already: https://github.com/sparkletown/sparkle/blob/staging/src/pages/Admin/Admin.tsx#L275

Are you sure there's an issue with the ordering?

Reply:
Didn't see that, kept the at the firebase level (has to be ascending instead of descending tho)